src/components/templates/Search/utils.ts

```diff
@@ -66,42 +66,72 @@ export function getSearchQuery(
       `(service.attributes.additionalInformation.categories:\"${categories}\")`
     : text || ''
 
+  if (searchTerm.startsWith('did:op:')) {
+    searchTerm = searchTerm.substring(7)
+  }
   // HACK: resolves the case sensitivity related to dataTokenInfo.symbol
   searchTerm = '*' + searchTerm.toUpperCase() + '*'
```

Comment:
stop adding `*` at the start and end of the string

Reply:
Added them for the cases user doesn't type all the search word. When searching by token symbol I found the following problem:
![Screenshot from 2021-06-07 17-22-43](https://user-images.githubusercontent.com/49017601/121035388-7ffa0880-c7b6-11eb-83d4-e8cb125a345f.png)
![Screenshot from 2021-06-07 17-22-55](https://user-images.githubusercontent.com/49017601/121035399-838d8f80-c7b6-11eb-8861-6da390b130ee.png)

Reply:
but it creates problems in other places. Searching for datatoken symbol will be fixed in https://github.com/oceanprotocol/aquarius/issues/492

Reply:
Oh, ok then I'll remove it. That was the only issue that `*` resolved.